src/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const getNewId = (items) => {
+  let maxId = 0;
+
+  for (const item of items) {
+    if (item.id >= maxId) {
+      maxId = item.id;
+    }
+  }
+
+  return maxId + 1;
+};
```

Comment:
`uuid` library can be used

Reply:
Sure, but I feel like getting item by id would be not the easiest thing then.